From c978dc00da1b2e7eaaf6f33194e2f49deaa68d5f Mon Sep 17 00:00:00 2001
Message-ID: <c978dc00da1b2e7eaaf6f33194e2f49deaa68d5f.1716242564.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1716242563.git.sharlatanus@gmail.com>
References: <cover.1716242563.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 19 May 2024 11:55:04 +0100
Subject: [PATCH 20/26] gnu: python-unyt: Fix version detection.

* gnu/packages/python-science.scm (python-unyt): Fix version detection.
[native-inputs]: Add python-setuptools-scm.

Change-Id: I96e7d2530b15299a39911bebf62b76a9191ea91b
---
 gnu/packages/python-science.scm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnu/packages/python-science.scm b/gnu/packages/python-science.scm
index 861badad2e..5e37177712 100644
--- a/gnu/packages/python-science.scm
+++ b/gnu/packages/python-science.scm
@@ -1860,7 +1860,7 @@ (define-public python-unyt
                              python-numpy
                              python-sympy))
     ;; Pint is optional, but we do not propagate it due to its size.
-    (native-inputs (list python-pint python-pytest))
+    (native-inputs (list python-pint python-pytest python-setuptools-scm))
     (home-page "https://unyt.readthedocs.io")
     (synopsis "Library for working with data that has physical units")
     (description
-- 
2.41.0

